packages/generator/src/sdk-metadata/__snapshots__/generation-and-usage.spec.ts.snap

```diff
@@ -1,23 +1,23 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`generation-and-usage create api specific usage for action import 1`] = `
-"import { dummyActionImport, dummyParamType } from './generated/undefined/action-imports';
+"import { dummyActionImport, dummyParamType } from './generated/undefined/operation-imports';
 
 const parameter: dummyParamType = { dummyParam1: 'dummyParam1' };
     
 const resultPromise = dummyActionImport(parameter).execute({ destinationName: 'myDestinationName' });"
 `;
 
 exports[`generation-and-usage create api specific usage for function import 1`] = `
-"import { dummyFunc, dummyParamType } from './generated/undefined/function-imports';
+"import { dummyFunc, dummyParamType } from './generated/undefined/operation-imports';
```

Comment:
[req] This really should be `operations` not `operation-imports`.

Reply:
[q] Why is there `undefined` in this path. I can see that it was there before, but it is kind of a red flag.